docs/luigi-core-api.md

```diff
@@ -884,10 +884,6 @@ Returns **[promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/
 
 -   **since**: 0.6.4
 
-<!-- add-attribute:class:warning -->
```

Comment:
I'm not sure why this note should be deleted. It seems still relevant. Maybe it was a bug when autogenerating the doc from the API .js file.